### PR TITLE
Remove label filter by clicking on the label is not working

### PR DESCRIPTION
**Pull request by aljesusg**
Fixes https://github.com/kiali/kiali/issues/3389

Backport: https://github.com/kiali/kiali-ui/pull/1996

### CODE REVIEW

**Comment by skondkar** [comment]
Tested on Kiali UI :   1.27.0 (9f9dea512f0a5fdc0f503821078540bab387b496)
I can still see the issue in my env while testing the PR. (Firefox 70 and Chrome 75)

Clicking on the label showing tooltip 'Remove label from Filters' does not remove the label filter from the list pages.

**Review by jshaughn — CHANGES_REQUESTED** [review]
Hi @aljesusg I Have not yet reviewed the code because the test failed, I see no difference in the behavior.  Clicking the non-highlighted label does filter correctly, but clickingt the highlighted label does not seem to have any effect.

**Comment by aljesusg** [comment]
@jshaughn @jotak I found the problem, we were matching "category" with the id => "Label" vs "label"

@skondkar please can you review it now ? 

Thanks

**Review by skondkar — APPROVED** [review]
Thanks @aljesusg 
Verified on Kiali UI:   1.27.0 (4b876c2e2a7504b4a592992bd39d10bc21fa3e8a)
Works as expected. Verified on overview page, applications/workloads/services list pages.

File: src/components/Filters/StatefulFilters.tsx [hunk 1]
```diff
@@ -234,7 +234,10 @@ export class StatefulFilters extends React.Component<StatefulFiltersProps, State
   };
 
   removeFilter = (id: string, value: string) => {
+    console.log('Remove ' + id + ' _ ' + value);
+    console.log(this.state.activeFilters.filters);
```
Comment (jshaughn) [inline src/components/Filters/StatefulFilters.tsx:238]: Need to remove.

File: src/components/Filters/StatefulFilters.tsx [hunk 2]
```diff
@@ -234,7 +234,10 @@ export class StatefulFilters extends React.Component<StatefulFiltersProps, State
...
     const updated = this.state.activeFilters.filters.filter(x => x.id !== id || x.value !== value);
+    console.log(updated);
```
Comment (jshaughn) [inline src/components/Filters/StatefulFilters.tsx:240]: Need to remove.

**Review by jshaughn — CHANGES_REQUESTED** [review]
@aljesusg Seems to be working now, just need to clean up some console logging.

File: src/components/Filters/StatefulFilters.tsx (hunk 2 above)
Comment (aljesusg) [reply src/components/Filters/StatefulFilters.tsx]: done @jshaughn I forgot remove it for this PR sorry and thanks

**Review by jshaughn — APPROVED** [review]
LG

**Pull request merged**